migrations/20200826214834-library.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  async up(db) {
+    const collectionList = await db.listCollections().toArray();
+
+    if (!collectionList.map(col => col.name).includes('reserves')) {
+      await db.createCollection('reserves');
+    }
+    if (!collectionList.map(col => col.name).includes('analytics')) {
+      await db.createCollection('analytics');
+    }
+  },
```

Comment:
I would still keep down() here just so that the command migrate-mongo down won't cause an error..

Reply:
@kilometerskang Do you mind testing if command migrate-mongo down would cause an error without the down() function?

Reply:
It does, so I'll just add it back